### PR TITLE
Release 1.16.0 rc1

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: resources/core/values.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ global:
     # - Change the value below to the branch from your fork if you want to point to documentation sources from your fork
     # - Change the value below to the release branch during the release
     # Example: clusterAssetGroupsVersion: "release-0.9"
-    clusterAssetGroupsVersion: master
+    clusterAssetGroupsVersion: release-1.16.0-rc1
```
Comment (dariadomagala) [inline resources/core/values.yaml:24]: here it's enough to have the "main" release version
```suggestion
    clusterAssetGroupsVersion: release-1.16
```

**Review by dariadomagala — CHANGES_REQUESTED** [review]

**Review by dariadomagala — APPROVED** [review]

**Review by magicmatatjahu — APPROVED** [review]

**Review by strekm — APPROVED** [review]
/meow

**Review by m00g3n — APPROVED** [review]

**Pull request merged**